doc/list.rst

```diff
@@ -1684,6 +1684,13 @@ List of Available Rules
    Part of rule set `@PhpCsFixer <./ruleSets/PhpCsFixer.rst>`_
 
    `Source PhpCsFixer\\Fixer\\ControlStructure\\NoUselessElseFixer <./../src/Fixer/ControlStructure/NoUselessElseFixer.php>`_
+-  `no_useless_nullsafe_operator <./rules/operator/no_useless_nullsafe_operator.rst>`_
+
+   There should not be useless ``null-safe-operators`` ``?->`` used.
```

Comment:
I am really impressed if someone was writing this code in their file

Reply:
I was as well when I found it in a project :see_no_evil:  : )    thanks for the merge!